style.css

```diff
@@ -0,0 +1,98 @@
+.body {
+  overflow-x: hidden;
+}
+
+.nav_container {
+  display: flex;
+  width: 100%;
+  margin-left: -25px;
+  margin-top: -8px;
+}
+
+.nav_container,
+.logo,
+.toggle {
+  display: flex;
+  padding: 20px;
+}
+
+.logo {
+  width: 25%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.logo img {
+  width: 100%;
+}
+
+.toggle img {
+  width: 50%;
+}
+
+.toggle {
+  width: 10%;
+  margin-left: auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.main_container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url("images/header-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.box1 {
+  width: 80%;
+  margin-top: 20%;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #172b4d;
+  padding: 24px;
+}
+
+.box2 {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-size: 25px;
+  vertical-align: top;
+  color: #344563;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 24px;
+}
```

Comment:
- [ ]  Please consider adjusting some of the styles properties you have here to match with your template on Figma. Try to check the image below and make changes so that your design match that of your template.
See here:
<img width="129" alt="Loom_fszdnqSP9x" src="https://user-images.githubusercontent.com/81468675/148226753-f8bbd3ce-811d-485a-b4cc-2225d1765d4d.png">